test/token/ERC20/extensions/ERC20FlashMint.test.js

```diff
@@ -93,60 +93,52 @@ contract('ERC20FlashMint', function (accounts) {
       await expectRevert.unspecified(this.token.flashLoan(receiver.address, this.token.address, MAX_UINT256, data));
     });
 
-    it('custom flash fee', async function () {
+    describe('custom flash fee & receiver', async function () {
       const receiver = await ERC3156FlashBorrowerMock.new(true, true);
```

Comment:
Hm, this doesn't look good. The describe function can't be async, and any contract deployments need to go inside in `beforeEach` blocks.